### ISSUE TITLE
No files being synced to dropbox despite logs saying so

### ISSUE THREAD

**Issue by siggeb** [matthewhadley/homeassistant-dropback#15]
Hey!
I installed this addon in october, and at that time, it did work, with files being sent to dropbox. 

I have been hunting down backup sizes recently, as I was up to 15GB, but now back to 4GB (some left to do). Does size affect the sync negatively? I searched for it, and found some issues with that and Google drive, somehow related to some python upgrades in HA, sound familiar?

Anyway, here are my logs, saying its being sent to dropbox, but nothing shows up. 

I did notice that the app key somehow had gotten bad, so I refreshed this. Before that I got this error in the logs:
[2024-07-19 07:25:52] ERROR: Got unexpected response from the API: Backup does not exist

But since I made the update, this error does now show up anymore. So only getting the below. Any hints?

[2024-07-20 04:32:51] INFO: Received input: sync
[2024-07-20 04:32:51] INFO: Deleting files older than 14 days...
[2024-07-20 04:32:51] INFO: Done with deletes
[2024-07-20 04:32:51] INFO: Syncing local files to Dropbox...
[2024-07-20 04:32:53] INFO: Synced /backup/d308a232.tar to /home_assistant_backup_sigge/addon_a0d7b954_nodered_18.0.3.tar on Dropbox
[2024-07-20 04:32:54] INFO: Synced /backup/ccd70340.tar to /home_assistant_backup_sigge/test.tar on Dropbox
[2024-07-20 04:32:55] INFO: Synced /backup/b24fd605.tar to /home_assistant_backup_sigge/2024-07-15.tar on Dropbox
[2024-07-20 04:32:57] INFO: Synced /backup/a3a4452f.tar to /home_assistant_backup_sigge/2024-07-16.tar on Dropbox
[2024-07-20 04:32:58] INFO: Synced /backup/f3481a32.tar to /home_assistant_backup_sigge/2024-07-17.tar on Dropbox
[2024-07-20 04:33:00] INFO: Synced /backup/4392a414.tar to /home_assistant_backup_sigge/2024-07-18.tar on Dropbox
[2024-07-20 04:33:03] INFO: Synced /backup/bc114ea8.tar to /home_assistant_backup_sigge/2024-07-19.tar on Dropbox
[2024-07-20 04:33:04] INFO: Synced /backup/e8705cfd.tar to /home_assistant_backup_sigge/2024-07-19.tar on Dropbox
[2024-07-20 04:37:55] INFO: Synced /backup/fa5313e7.tar to /home_assistant_backup_sigge/2024-07-20.tar on Dropbox
[2024-07-20 04:37:55] INFO: Syncing done
[2024-07-20 04:37:55] INFO: Listening for input via stdin service call...

**Comment by matthewhadley** [comment]
Looking at the timestamps of the log, there are only a few seconds between them except the last one [2024-07-20 04:33:04] to [2024-07-20 04:37:55]. This is consistent with checking a file is already there (so no need to upload) and then finding the last file needs to be upload and so doing that. You're totally sure that there are no files in your dropbox folder? (there have been cases where uses have linked a different dropbox account, as they have multiple).

I have not seen this error before:
```ERROR: Got unexpected response from the API: Backup does not exist```

**Comment by matthewhadley** [comment]
Closing due to inactivity

**Comment by siggeb** [comment]
Hey, 
Sorry for the silence here, family things made me not able to look into this. But just last week I noticed in the IOS Dropbox app that there were backups created. But again, looking in the web app and locally, I could not see them. 

What has happened then? Well, you do describe it in the docs, but it is not so obvious that I notived before. BUT...

When creating that App scoped token, the "App folder" is created under the <dropbox root folder>/Apps/<App folder>
But since I was darn sure that when I set this up, I created a folder in my root, that had this scope set to it. But now I realize when verifying this setup, I was looking in the iOS app that is smart enough to just show whatever folder had the latest updates, without me realizing it is in the Apps folder...

So while I cannot blame you, since its mentioned in the docs, but it was not completely obvious to me =P

And since I have been struggling with 5GB sized backups, I thought this was the issue....
